lib/krikri/index_querier.rb

```diff
@@ -0,0 +1,17 @@
+module Krikri
+  # Queries the Search Index
+  class IndexQuerier
```

Comment:
Ah, I see what you're trying to do. At least for the first two, maybe we should address it this way: https://github.com/projectblacklight/blacklight/blob/master/lib/blacklight/search_helper.rb#L7-L16